clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/DownloadStrategyResolver.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import { SettingsService } from "../../Settings";
+import { CopybookName } from "../CopybookDownloadService";
+import { CopybookDownloader } from "./CopybookDownloader";
+import { CopybookDownloaderForDsn } from "./CopybookDownloaderForDsn";
+import { CopybookDownloaderForUss } from "./CopybookDownloaderForUss";
+
+/**
+ * Copybook downloader from MVS or USS using Zowe Explorer based on configuration
+ */
+export class DownloadStrategyResolver implements CopybookDownloader {
```

Comment:
Why does the class implement the `CopybookDownloader` interface? It does not seem to be used via the interface anywhere in the code.

Reply:
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/c597da87df03ac9820fe5c7eab2f607d3a6d6da4/clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts#L28

Reply:
True, but you only every instantiate DownloadStrategyResolver as the value for it. It is also never undefined.

Reply:
updated